components/layout/MainHero.module.css

```diff
@@ -18,4 +18,18 @@
     text-align: center;
     margin: 0 auto;
     font-size:2vw;
-}
\ No newline at end of file
+}
+
+.image {
+    animation: fade-in 1s forwards 1;
+  }
+  
+  @keyframes fade-in {
+    0% {
+      opacity: 0;
+    }
```

Comment:
The indentation here is not very consistent. If you find it difficult to keep it consistent by hand, consider activating automatic formatting in your editor.